README.md

```diff
@@ -32,7 +32,7 @@ $ tsd query jquery.*
 $ tsd query "jquery.*"
 
 #install all definitions from tsd.json
-$ tsd install
+$ tsd reinstall
```

Comment:
Hmm, so why do I get this error message when I run tsd install?
pass at least one query pattern
VError: pass at least one query pattern

With reinstall it works as excepted.